cbl_flutter:installBinaries

Using vs_code on Windows 10. Simply can't build a project.
From the doc page it seem like I need the cbl_flutter dependency. Is there any other way to get the binaries and where do I put them?

build output:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':cbl_flutter:installBinaries'.
> A problem occurred starting process 'command './tool/install_binaries.sh''

```

--- a/packages/cbl_flutter/android/build.gradle
+++ b/packages/cbl_flutter/android/build.gradle
@@ -1,3 +1,5 @@
+import org.apache.tools.ant.taskdefs.condition.Os
+
 group 'com.terwesten.gabriel.cbl_flutter'
 version '1.0-SNAPSHOT'
 
@@ -47,7 +49,12 @@ dependencies {
 
 task installBinaries(type: Exec) {
     workingDir file("$projectDir/..")
-    commandLine './tool/install_binaries.sh', 'android'
+
+    if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+        commandLine 'cmd', '/c', 'C:/Program Files/Git/bin/bash', './tool/install_binaries.sh', 'android'
+    } else {
+        commandLine './tool/install_binaries.sh', 'android'
+    }
 }
 
 tasks.preBuild.dependsOn(installBinaries)

--- a/packages/cbl_flutter/tool/install_binaries.sh
+++ b/packages/cbl_flutter/tool/install_binaries.sh
@@ -11,18 +11,33 @@
 
 set -e
 
-# Skip installation if environment variable is set
+# Skip installation if environment variable is set.
 if [ -n "$CBL_FLUTTER_SKIP_INSTALL_BINARIES" ]; then
     echo "cbl_flutter: Skipping install of published binaries"
     exit 0
 fi
 
-# Setup parameters
+# The directory of this script.
 dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# The directory of the 'cbl_flutter' package. 
 pkgDir="$(cd "$dir/.." && pwd)"
+
+# The directory of the 'cbl_flutter' package formatted properly for the 
+# current system.
+pkgDirSys=
+case $(uname -s) in
+    CYGWIN*)    pkgDirSys=$(cygpath -m "$pkgDir");;
+    MINGW*)     pkgDirSys=$(cygpath -m "$pkgDir");;
+    *)          pkgDirSys=pkgDir
+esac
+
+# The names of all platforms for which binaries are available.
 allPlatforms=(android apple)
 
 platformArg="$1"
+
+# The names of the platforms for which to install binaries.
 platforms=
 if [ "$platformArg" = "all" ]; then
     platforms=("${allPlatforms[@]}")
@@ -59,10 +74,10 @@ function installBinariesForPlatform() {
     cat >pubspec.yaml <<-EOF
 name: tmp
 environment:                                                            
-  sdk: '>=2.12.0-0 <3.0.0' 
+  sdk: '>=2.12.0 <3.0.0' 
 dependencies:
     cbl_flutter:
-        path: "$pkgDir"
+        path: "$pkgDirSys"
     cbl_native: any
 EOF
 

--- a/packages/cbl_native/lib/cbl_native.dart
+++ b/packages/cbl_native/lib/cbl_native.dart
@@ -3,6 +3,7 @@ import 'dart:io';
 import 'dart:math';
 
 import 'package:http/http.dart' as http;
+import 'package:path/path.dart';
 
 /// GitHub repository data.
 class GitHubRepo {
@@ -117,9 +118,9 @@ class CblNativeBinaries {
       'tar',
       [
         '-xzf',
-        archiveUri.path,
+        fromUri(archiveUri),
         '-C',
-        installDir.path,
+        fromUri(installDir.uri),
       ],
     );
 

--- a/packages/cbl_native/pubspec.yaml
+++ b/packages/cbl_native/pubspec.yaml
@@ -10,6 +10,7 @@ environment:
 dependencies:
   collection: ^1.15.0
   http: ^0.13.0
+  path: ^1.8.0
   yaml: ^3.0.0
 
 dev_dependencies:

